### PR TITLE
Create the-journal-of-the-indian-law-institute.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/84206/style-request-the-journal-of-the-indian-law-institute-annual-survey-of-indian-law

### CODE REVIEW

File: the-journal-of-the-indian-law-institute.csl [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+      <name and="text" delimiter=", "/>
+    </names>
+  </macro>
+  <macro name="recipient-short">
+    <names variable="recipient">
+      <label form="verb" prefix=" " suffix=" "/>
+      <name form="short" and="text" delimiter=", "/>
+    </names>
+  </macro>
+  <macro name="contributors-short">
```
Comment (bwiernik) [inline the-journal-of-the-indian-law-institute.csl:177]: This macro isn't used anywher.

File: the-journal-of-the-indian-law-institute.csl [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <text variable="title" quotes="true"/>
+      </else-if>
+      <else>
+        <text variable="title" form="long" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="title-short">
```
Comment (bwiernik) [inline the-journal-of-the-indian-law-institute.csl:50]: This macro isn't used anywhere.

File: the-journal-of-the-indian-law-institute.csl [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+              <date-part name="month" prefix=" "/>
+              <date-part name="day" prefix=" "/>
+              <date-part name="year" prefix=", "/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locators-subsequent">
```
Comment (bwiernik) [inline the-journal-of-the-indian-law-institute.csl:281]: Macro not used anywhere.

**Review by bwiernik — CHANGES_REQUESTED** [review]

**Comment by rmzelle** [comment]
@POBrien333, thanks!

**Pull request merged**